code/web/src/modules/user/api/state.js

```diff
@@ -7,12 +7,20 @@ export const userInitialState = {
   error: null,
   isLoading: false,
   isAuthenticated: false,
-  details: null
+  details: null,
+  isSurveyCompleted: false
 }
 
 // State
 export default (state = userInitialState, action) => {
   switch (action.type) {
+    case ASIGN_STYLE:
```

Comment:
We can change the naming of this to 'ASSIGN'

Reply:
change it from `ASIGN` to just `ASSIGN` or `ASSIGN_STYLE`?